arduino-hal/Cargo.toml

```diff
@@ -37,4 +37,5 @@ optional = true
 features = ["disable-device-selection-error"]
 
 [dependencies.avr-device]
-version = "0.3"
+git = "https://github.com/Rahix/avr-device"
+rev = "92e4d5f"
```

Comment:
Tip: The best way to substitute `avr-device` with a local version for all crates in `avr-hal` is via a patch in the top-level `Cargo.toml` file (where the workspace is defined):

```toml
[patch.crates-io]
avr-device = { git = "https://github.com/Rahix/avr-device", rev = "92e4d5f" }
```

Then you won't have to touch every single Cargo.toml across the repository and it is also less work to undo once everything is ready.

Reply:
Ah, TIL.